src/www/ui_components/app-root.html

```diff
@@ -461,50 +466,50 @@
           type: Object,
           readonly: true,
           value: {
-            am: {id: "am", dir: "ltr"},
-            ar: {id: "ar", dir: "rtl"},
-            bg: {id: "bg", dir: "ltr"},
-            ca: {id: "ca", dir: "ltr"},
-            cs: {id: "cs", dir: "ltr"},
-            da: {id: "da", dir: "ltr"},
-            de: {id: "de", dir: "ltr"},
-            el: {id: "el", dir: "ltr"},
-            en: {id: "en", dir: "ltr"},
-            "es-419": {id: "es-419", dir: "ltr"},
-            fa: {id: "fa", dir: "rtl"},
-            fi: {id: "fi", dir: "ltr"},
-            fil: {id: "fil", dir: "ltr"},
-            fr: {id: "fr", dir: "ltr"},
-            he: {id: "he", dir: "rtl"},
-            hi: {id: "hi", dir: "ltr"},
-            hr: {id: "hr", dir: "ltr"},
-            hu: {id: "hu", dir: "ltr"},
-            id: {id: "id", dir: "ltr"},
-            it: {id: "it", dir: "ltr"},
-            ja: {id: "ja", dir: "ltr"},
-            ko: {id: "ko", dir: "ltr"},
-            km: {id: "km", dir: "ltr"},
-            lt: {id: "lt", dir: "ltr"},
-            lv: {id: "lv", dir: "ltr"},
-            nl: {id: "nl", dir: "ltr"},
-            no: {id: "no", dir: "ltr"},
-            pl: {id: "pl", dir: "ltr"},
-            "pt-BR": {id: "pt-BR", dir: "ltr"},
-            ro: {id: "ro", dir: "ltr"},
-            ru: {id: "ru", dir: "ltr"},
-            sk: {id: "sk", dir: "ltr"},
-            sl: {id: "sl", dir: "ltr"},
-            sr: {id: "sr", dir: "ltr"},
-            "sr-Latn": {id: "sr-Latn", dir: "ltr"},
-            sv: {id: "sv", dir: "ltr"},
-            th: {id: "th", dir: "ltr"},
-            tr: {id: "tr", dir: "ltr"},
-            uk: {id: "uk", dir: "ltr"},
-            ur: {id: "ur", dir: "rtl"},
-            vi: {id: "vi", dir: "ltr"},
-            zh: {id: "zh", dir: "ltr"},
-            "zh-CN": {id: "zh-CN", dir: "ltr"},
-            "zh-TW": {id: "zh-TW", dir: "ltr"},
+            am: {id: "am", name: "አማርኛ", dir: "ltr"},
+            ar: {id: "ar", name: "العربية", dir: "rtl"},
+            bg: {id: "bg", name: "Български", dir: "ltr"},
+            ca: {id: "ca", name: "Català", dir: "ltr"},
+            cs: {id: "cs", name: "Česky", dir: "ltr"},
+            da: {id: "da", name: "Dansk", dir: "ltr"},
+            de: {id: "de", name: "Deutsch", dir: "ltr"},
+            el: {id: "el", name: "Ελληνικά", dir: "ltr"},
+            en: {id: "en", name: "English", dir: "ltr"},
+            "es-419": {id: "es-419", name: "Español (Latinoamérica)", dir: "ltr"},
```

Comment:
nit: let's just call it "Español" since we don't support other variants.

Reply:
Done